### DISCUSSION TITLE
What first column with "e" or "." or "!" means? is it possible to disable it?

### DISCUSSION THREAD

**Discussion by user9931** [dundee/gdu#290] [Q&A] [answered]
What first column with "e" or "." or "!" means? is it possible to disable it?

Tried to remove from -n mode with awk, and its not very simple as its cannot be detected as -column- because some lines have this characters, most are not, only regex

**Comment by dundee** [comment, marked as answer]
These are flags that say if there was some error or if the file is somehow special.

See https://github.com/dundee/gdu?tab=readme-ov-file#file-flags for more details.

Currently it's not possible to disable it, but it might be possible to add it as a configuration option.
